static/app/components/replays/table/replayTable.tsx

```diff
@@ -153,14 +115,6 @@ const ReplayTableWithColumns = styled(
   }
 `;
 
-function getGridTemplateColumns(columns: readonly ReplayTableColumn[]) {
-  return `grid-template-columns: ${columns
-    .map(column =>
-      column === ReplaySessionColumn ? 'minmax(150px, 1fr)' : 'max-content'
-    )
```

Comment:
Moved this custom `column === ReplaySessionColumn` check into the `ReplayTableColumn` fields directly. Some inheritance/OOP in here.